packages/cli-platform-android/src/commands/runAndroid/runOnAllDevices.ts

```diff
@@ -148,8 +148,8 @@ function createInstallError(error: Error & {stderr: string}) {
   }
 
   return new CLIError(
-    `Failed to install the app. ${message}`,
-    message.length > 0 ? undefined : error,
+    `Failed to install the app.${message || ''}`,
```

Comment:
Spacing bug? Currently this would print

> `Failed to install the app.Make sure you have an Android emulator running or a device connected.`

```suggestion
    `Failed to install the app.${message ? (' ' + message) : ''}`,
```